Recon/python/simpleTrigger.py

```diff
@@ -40,7 +40,10 @@ def __init__(self, name, beamEnergy) :
         super().__init__(name,'recon::TriggerProcessor','Recon')
           
         self.beamEnergy = beamEnergy
-        self.thresholds = [ self.beamEnergy/4000.*1500.0, self.beamEnergy/4000.*1000. + self.beamEnergy, self.beamEnergy/4000.*500. + 2*self.beamEnergy, self.beamEnergy/4000.*100. + 3*self.beamEnergy ]
+        if (self.beamEnergy == 4000.): 
+            self.thresholds = [ self.beamEnergy/4000.*1500.0, self.beamEnergy/4000.*1000. + self.beamEnergy, self.beamEnergy/4000.*500. + 2*self.beamEnergy, self.beamEnergy/4000.*100. + 3*self.beamEnergy ]
```

Comment:
At this point, I'd also not have this as a function of energy but have the direct numbers. The reason for having the energy there was because we wanted to hand both energy case in one line, but now that we are moving away from it...
```suggestion
            self.thresholds = [ 3000, 10000, 17000, 24200 ]
```

Reply:
If we were to have different thresholds for 20 vs. 34 layers, would we just use one set of thresholds for <= 20 layers and another for > 20 layers? Also, is this the most recent version of Megan's thresholds? [https://indico.fnal.gov/event/51366/contributions/225984/attachments/148282/190432/MeganLoh_multielectronTriggering.pdf](url)

Reply:
> one thing that could be interesting is to add different defaults for 20 or 34 layers.

This, we should do in another PR. 

As for the numbers for 4 GeV, I have the same request to keep 1e as it is for consistency. Based on that talk, 2e should be at 5, 3e at 8.2, 4e at 11.8, right? @EBerzin will you push that change?